### PR TITLE
Configure updated `journald_monitor`

### PR DESCRIPTION
**Pull request by anton-ryzhov**
Configure built-in `journald_monitor`

### CODE REVIEW

File: kube_log_watcher/templates/scalyr.json.jinja2 [hunk 1]
```diff
@@ -46,18 +46,24 @@
             {% endif %}
             {% if monitor_journald.attributes %}
             "attributes": {
+                "foo": "bar1",
```
Comment (astarnell) [inline kube_log_watcher/templates/scalyr.json.jinja2]: Why this `foo` attribute?

Comment (anton-ryzhov) [reply kube_log_watcher/templates/scalyr.json.jinja2]: This a draft PR

**Comment by anton-ryzhov** [comment]
:+1:

**Comment by anton-ryzhov** [comment]
:+1:

**Comment by astarnell** [comment]
:+1:

**Pull request merged**